proto/pocket/migration/tx.proto

```diff
@@ -187,6 +192,9 @@ message MsgClaimMorseSupplier {
   // See: https://dev.poktroll.com/operate/configs/supplier_staking_config#staking-types.
```

Comment:
If we're not ordering the indecies from top to bottom, I highly recommend following the `Next Free Index` comment pattern.

![Screenshot 2025-04-10 at 1 35 30 PM](https://github.com/user-attachments/assets/3d72b9fc-10ee-40e4-91a0-ddf5f67908a4)

Reply:
This is something that I would argue we should automate. I'm happy to do it and/or open an issue for it in the meantime. I think it should be pretty straightforward given that we've already done this sort of thing previously for ensuring the presence of the `stable_marshaler_all` protobuf option on all protobufs (see: [tools/scripts/protocheck/cmd](https://github.com/pokt-network/poktroll/blob/main/tools/scripts/protocheck/cmd/unstable.go)).

Reply:
- #1193

Reply:
After we ship everything, we're going to have a lot of time to automate things. As long as it's after we ship everything, no objections.

![Screenshot 2025-04-11 at 9 49 31 AM](https://github.com/user-attachments/assets/e1710bb1-f41b-4b76-bcd9-15f938df8128)